src/main/java/org/fundacionjala/coding/escarleth/Kata4.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.escarleth;
+
+/**
+ * Powered by Escarleth Ledezma Q.
+ * Kata4 is a class that contains a digitalRoot method
+ */
+public class Kata4 {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)